README.md

```diff
@@ -73,9 +73,42 @@ development.
 - [IBC Go implementation][ibc-go]
 - [Protobuf definitions in Rust][ibc-proto-rs]
 
+## `ibc-rs` in the wild
+
+The IBC protocol is integrated in numerous networks and projects. The `ibc-rs` implementation specifically
+is adopted for IBC integration, testing, or development in the following use-cases.
+
+|  # | Integration use-case |     Team      | Used-Modules |                 Links                 |          Phase           |
+|---:|----------------------|:-------------:|--------------|:-------------------------------------:|:------------------------:|
+|  1 | Solana               |  Composable   | all          |    [Picasso Network][pica-network]    |    :white_check_mark:    |
```

Comment:
If you want to be specific with the use cases, 

- Solana IBC implementation
- Ethereum IBC relayer
- Polkadot SDK IBC implementation

Reply:
Suggested language change for the introduction:

The [IBC protocol](https://ibcprotocol.dev/) is integrated in numerous networks and projects. The `ibc-rs` implementation, in particular, is used for IBC integration, testing, or development for some of the most technically compelling use cases.

Reply:
@JafarAz can you tell us which ibc-rs components you are using in each of these integrations?

Reply:
@rnbguy we're using the core library, Tendermint client and transfer module